helper/league.py

```diff
@@ -27,4 +27,36 @@ def display_team_info(team : LazyList) -> str:
         players.reverse()
         for player in players:
             info += LeagueClientHelper.display_summoner_info(player)
+        return info
+    
+    @staticmethod
+    def display_summoner_pregame_info(player : Participant) -> str:
+        name = player.summoner.name
+        champion = player.champion.name
+        soloRank = ""
+
+        flexRank = ""
+        try:
+            rank = [player.summoner.ranks[Queue.ranked_solo_fives].tier][0]
+            divison = [player.summoner.ranks[Queue.ranked_solo_fives].division][0]
+            soloRank = str(rank) + " " + str(divison)
+        except:
```

Comment:
Is there a better way you can handle the not ranked case rather than expecting an exception? Just wondering

Reply:
So if the summoner does not have a rank, the api throws an exception. That is why I am using "try except". If you have better solution to handle this case I can change it to that.